src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/testhosts/UtilTestServiceHost.cs

```diff
@@ -0,0 +1,31 @@
+﻿//  Copyright (c) Microsoft Corporation.  All Rights Reserved.
+using System;
+using System.ServiceModel;
+using System.ServiceModel.Activation;
+using System.ServiceModel.Channels;
+
+namespace WcfService
+{
+    public class UtilTestServiceHostFactory : ServiceHostFactory
```

Comment:
Not strictly required for checkin, but Util.cs should be renamed and UtilTestServiceHost should be renamed.  The name is too general and doesn't describe what it does.